.github/docs/openapi3.txt

```diff
@@ -3,18 +3,21 @@ const TypeArray = "array" ...
 const FormatOfStringForUUIDOfRFC4122 = `^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$` ...
 const SerializationSimple = "simple" ...
 var SchemaErrorDetailsDisabled = false ...
+var ErrInvalidVersion ...
```

Comment:
What's your rationale to export this?

Reply:
So one can check at loading time he got an invalid error version, not some other kind of error.